server/server.js

```diff
@@ -63,6 +67,8 @@ app.prepare()
         // use routes
         server.use(userInViews());
         server.use('/api/users', users);
+        server.use('/api/subjects', subjects);
+        server.use('/api/contents', contents);
```

Comment:
define all the different routes in your `routes/index.js` not in this server file. then simiply import the routes into this file and do the following
```
const router = require('./routes);
...
server.use('/', router);
```